vue-components/src/components/DateInput.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-DateInput' ]"
+	>
+		<InputWithExtender
+			:label="label"
+			:value="value"
+			:error="error"
+			:placeholder="placeholder"
+			:disabled="disabled"
+			@input="$emit( 'input', $event )"
+		>
+			<template v-slot:default>
+				<p class="wikit-DateInput__IntroText">
+					{{ resultsIntroText }}
+				</p>
+				<p class="wikit-DateInput__ParsedValue" v-if="parsedValue">
+					<b>{{ parsedValue }}</b>
+				</p>
+				<slot v-if="!value" name="prompt" />
+				<BouncingDots v-if="!parsedValue && value" type="small" />
+				<div class="wikit-DateInput__CalendarNotice" v-if="calendarNotice && parsedValue">
+					<Icon type="info-outlined" size="small" />
```

Comment:
```suggestion
					<Icon type="info-filled" size="small" />
```

Reply:
my bad! I realized we should be using the icon that accompanies notice messages.